test/cli/cli.test.js

```diff
@@ -586,6 +862,14 @@ describe('CLI', () => {
   });
 
   it('should generate correct cli flags', (done) => {
+    const isMacOS = process.platform === 'darwin';
```

Comment:
What is problem? Not all output? It means we have bug somewhere...

Reply:
I think problem with exec https://github.com/nodejs/node/issues/19218

Reply:
Yes, good catch :+1: